kafka/src/test/java/io/specmesh/kafka/KafkaAPISpecFunctionalTest.java

```diff
@@ -50,20 +51,19 @@ public class KafkaAPISpecFunctionalTest {
 
     private AdminClient adminClient;
 
-    @BeforeEach
-    public void createAllTheThings() {
+    public KafkaAPISpecFunctionalTest() throws ExecutionException, InterruptedException, TimeoutException {
```

Comment:
As above, this would be better done with bring up the docker containers once per class, rather than once per test.

Reply:
yep - got it